weatherman.py

```diff
@@ -0,0 +1,189 @@
+import calendar
+import csv
+import os
+import sys
+
+
+class temperatureTracker():
```

Comment:
Class names should be in `CamelCase`

Reply:
Why are we defining the class with empty parameters?

Reply:
If you prefer to move into classes, make sure all your code goes that way. You can't have a mixture of both because code looks odd this way. Let's convert all concerns into relevant classes and use them in object oriented way.